src/components/chat.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import {FormControl, InputGroup, Container, Nav, Navbar, NavDropdown} from "react-bootstrap";
+
+import logo192 from '../assets/images/logo192.png'
+
+
+class CustomNavbar extends React.Component {
```

Comment:
I actually separated the CustomNavbar from the chat page as the navbar can be reused multiple times. I think we should separate the concept of components and pages. It seams reasonable that components are objects that are reused multiple times such as a navbar and pages are only used once (in this case, chat page).